### ISSUE TITLE
RFC2: Board of Directors

### ISSUE THREAD

**Issue by k8hertweck** [carpentries/2017Merger#2]
View RFC2 [here](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC2.md)

**Q1: Do you agree with above described responsibilities of the Board of Directors of The Carpentries organisation?** Specifically:

Q1a: Are there any responsibilities missing?

Q1b: Is there anything for which the Board should not be responsible?

**Comment by JasonJWilliamsNY** [comment]
Q1: I am not too picky, but it is hard to interpret what nomination (with community input) means. Will there just be a general call for nominations? As long as the process is transparent this is good. 

It would be interesting to choose appointed board members after an election to better control composition of the board. In the end I like the proposed system because you have the ability to draw in the expertise/experience needed. As always, getting a diverse board can be difficult, and this can be hard on the membership too if folks only know a few members. I think through mentoring it is easier for folks to get their faces out their and be know tho. 

I also think that finances need to be separated from the board, in that accounting details should be a staff job, with the board in the decision making and auditing roles. Perhaps the first task is for the Constitution/by-laws to be drafted and ratified, cause little details like this are missing or unclear now I think (which is OK at this point).

**Comment by pitviper6** [comment]
>Appointed members ensure that the Board has the expertise required for strategic goals

Having been on a BOD for a professional association, this bit can be overlooked and when it is, things can get wacky. So, I'm glad this has been specifically mentioned.

Another way of increasing the diversity of boards is communicating early and often that you don't have to be asked to step up to leadership. You can volunteer - you SHOULD volunteer, if you want to contribute in that way. I can't tell you how hard it is to reverse the perception about having to be asked.

Also, agreeing with @JasonJWilliamsNY about finances being separate and a staff responsibility that then submits financial reports to the board (which I think was the way it was put in R1).

**Comment by rgaiacs** [comment]
The "nomination" seats was never very clear to me. Who will make the nomination? Who will select members among the nominated? How long is going to be the terms for nominated seats?

I also would prefer a majority of seats elected by the community.

**Comment by kcranston** [comment]
You are correct to note that the nomination process, term length, etc is not yet fully specified. That level of detail will come with the writing of bylaws (informed by the comments we get here). Agree on finances being a staff role, with oversight by the BoD. 

@rgaiacs - the majority appointed + minority elected composition of the board has already been voted on by both DC and SC steering committees, so that isn't going to change in the short term.

**Comment by npch** [comment]
I think it's good that the Board of Directors has the governance / steering / auditing role. This gives a clear separation between those who are undertaking the work (staff and volunteers with specific roles) and those who are checking that the work helps the organisation to achieve its goals without compromising its values.

I would suggest that the nominations process makes it explicit that appointed members are sought to balance composition, diversity (if legally allowed) and expertise on the board. 

An example of an *election* process which takes into account balancing is RDA's TAB election process: https://www.rd-alliance.org/call-nominations-rda-technical-advisory-board-tab

**Comment by npch** [comment]
A fairly normal responsibility for a Board of Directors of an organisation like this is selecting and appointing the Executive Director when necessary. So perhaps the item should be extended to read:

- Selecting, appointing and terminating the Executive Director
- Supporting and providing continual feedback relating to performance evaluation for the Executive Director

**Comment by npch** [comment]
It might be good to be explicit that the Board of Directors are:

- Accountable to the Carpentries community for the organisation's performance
- Responsible for setting the salaries, compensation and benefits of senior staff
- Responsible for approving annual _(or whatever is the appropriate time period)_ budgets

**Comment by npch** [comment]
It should be clarified what the legal responsibilities of board members are (including jurisdiction). However I appreciate that this might not be possible until the bylaws are drafted and reviewed.

**Comment by tracykteal** [comment]
As a fiscally sponsored project, and not our own 501(c)3, there are not legal obligations of board members. However, it is important to be clear about responsibilities. Definitely the oversight of the Executive Director will need to be in the Board responsibilities, as it currently is for SWC and DC's respective Steering Committees.

**Comment by gvwilson** [comment]
I don't think it's possible to assess this proposal without knowing who is able to nominate candidates for the nominated positions on the board. I agree with @kcranston that term length etc. can be left to bylaws, but there is a big difference between "ED nominates", "community members nominate", and "Membership Council nominates".

**Comment by npch** [comment]
@tracykteal agreed that we are talking about responsibilities here rather than obligations/accountability, due to the fiscally sponsored direct project arrangement.

I think it's important to say something about the relationship with the Sponsoring Organization. For instance, will they have a position on the Carpentries Board? If not, what is the governance mechanism to ensure that the aims of the Carpentries (as a fiscally sponsored project) remain aligned to the aims of the sponsoring organisation. Do any category of decisions made by the Board have to be ratified by the sponsoring organisation's Board (for instance, anything which might have significant legal repercussions like the termination of a senior staff contract)?

**Comment by kcranston** [comment]
@gvwilson - many details, including the nomination process, haven't yet been discussed within the merger committee. This lack of detail is intentional. We are looking for suggestions through this RFC about how we should best structure the process.

**Comment by kcranston** [comment]
I'd like to hear some suggestions to the question of "who can nominate someone to the board?". Community members? Other board members? Membership Council nominates? Other?

**Comment by npch** [comment]
As an example, under UK law, the Board as a whole is responsible for the appointment of all directors to the board, whether by election or nomination/appointment, though the process (as defined in the equivalent of bylaws) may allow for nominations to be sought through many mechanisms. 

As a strawperson argument, here are some potential mechanisms for nominations:
- community members are encouraged to self-nominate where they fit skills or experience identified by a Skills Audit (however this is really what the open elections should do)
- community members are encouraged to nominate people who represent specific (and currently unrepresented) interest groups, networks, viewpoints or perspectives that would be valuable for the organisation 
- membership partners are asked to nominate people (this is similar to major shareholders having an active stake in the running of an organisation they invest in)
- the organisation runs an external recruitment process, using a specialist to identify suitable appointees who will provide a different perspective

In each case, recommendations of people would be passed to the board for scrutiny. At that point I could see two potential ways of formalising appointments: the board has the power to formalise these appointments directly, or the board co-opts the recommended candidates to the board and they are formally appointed at a vote at the AGM.

**Comment by gvwilson** [comment]
I would like member organizations to nominate candidates for the Board:

1. It's a tangible benefit of membership (without being an obligation).
2. It gives member orgs a clearly-defined role in governance (albeit indirect).
3. Member orgs are likely to be cognizant of the challenges and needs in running an organization of this scale.

If this is adopted, the Board and/or ED could set out requirements annually for specific skills the Carpentry org needs for the coming year(s) so that member orgs would know what to focus on.
